src/core/worker.js

```diff
@@ -436,6 +437,55 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
+          return obj;
+        } else if (isName(obj)) {
+          return obj;
```

Comment:
`Name` objects are usually not transferable via `postMessage`. Return `{name: obj.name};` instead.